tools/bumper/git_fakes.go

```diff
@@ -344,3 +345,31 @@ func createCommitWithLightweightTag(w *git.Worktree, repo *git.Repository, tagCo
 func getFakePrWithTitle(prTitle string) *github.NewPullRequest {
 	return &github.NewPullRequest{Title: &prTitle}
 }
+
+func fileExists(filename string) bool {
+	info, err := os.Stat(filename)
+	if os.IsNotExist(err) {
+		return false
+	}
```

Comment:
In case of error different from `IsNotExist` it will return true.

Reply:
Done